client/src/modules/shopping-list/components/InputBar/model/actions.js

```diff
@@ -1,14 +1,16 @@
 import { ENDPOINT_URL } from 'common/constants/variables';
 import { postData } from 'common/utils/fetchMethods';
-
-// Action types
-export const ADD_ITEM = 'ADD_ITEM';
-export const ADD_ITEM_FAILURE = 'ADD_ITEM_FAILURE';
-export const ADD_ITEM_SUCCESS = 'ADD_ITEM_SUCCESS';
+import { ItemActionTypes } from '../enum';
 
 // Action creators
-export const addItemError = err => ({ type: ADD_ITEM_FAILURE, err });
-export const addItemSuccess = item => ({ type: ADD_ITEM_SUCCESS, item });
+export const addItemError = err => ({
+  type: ItemActionTypes.ADD_ITEM_FAILURE,
```

Comment:
addItemSuccess -> ADD_ITEM_SUCCESS
addItemError -> ADD_ITEM_FAILURE

Reply:
What you mean?